tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -809,12 +809,10 @@ describe("Service Form Modal", function() {
           });
         });
 
-        it('should disable the "Advanced Settings" section', function() {
-          cy.contains("Advanced Settings").parent().within(function() {
-            cy
-              .get("label.form-control-toggle.disabled")
-              .should("have.length", 2);
-          });
+        it('should disable the "Grant Runtime Privileges" checkbox', function() {
```

Comment:
I believe this should say `should be disabled` instead `should disable` which make you think the test is an action to disable but it's only confirming.